<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(205,219,228)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>330</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>911</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>SAIE User</string>
     </attribute>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>421</width>
        <height>501</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> border: 1px solid black;
border-radius: 10px;
background-color:rgb(81,122,153)</string>
      </property>
      <widget class="QPushButton" name="open_file_btn">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>40</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white</string>
       </property>
       <property name="text">
        <string>Open File</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white</string>
       </property>
       <property name="text">
        <string>Files</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_input">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>321</width>
         <height>251</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> border: 1px solid black;
border-radius: 10px;
background-color:rgb(255,255,255)</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>20</y>
        <width>431</width>
        <height>501</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
 border: 1px solid black;
border-radius: 10px;
background-color:rgb(81,122,153)</string>
      </property>
      <widget class="QListWidget" name="listWidget_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>351</width>
         <height>351</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
border-radius: 10px;</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_output">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>351</width>
         <height>251</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> border: 1px solid black;
border-radius: 10px;
background-color:rgb(255,255,255)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>47</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> border: 1px solid black;
border-radius: 10px;
background-color:rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string>Resultat</string>
       </property>
      </widget>
      <widget class="QPushButton" name="show_Button">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>360</y>
         <width>75</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white</string>
       </property>
       <property name="text">
        <string>show</string>
       </property>
      </widget>
      <widget class="QPushButton" name="modify_btn">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>360</y>
         <width>75</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white</string>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="retrain_btn">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>360</y>
         <width>75</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white</string>
       </property>
       <property name="text">
        <string>Retrain</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>SAIE Admin</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_vect_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>391</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white</string>
      </property>
     </widget>
     <widget class="QPushButton" name="vectoriel_search_button_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>50</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white</string>
      </property>
      <property name="text">
       <string>Ajouter un corpus</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>301</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Recherche</string>
     </attribute>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>-190</x>
        <y>580</y>
        <width>120</width>
        <height>80</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="widget_5" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>29</y>
        <width>431</width>
        <height>501</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> border: 1px solid black;
border-radius: 10px;
background-color:rgb(81,122,153)</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_vect">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>391</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white</string>
       </property>
      </widget>
      <widget class="QPushButton" name="vectoriel_search_button">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>391</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white</string>
       </property>
       <property name="text">
        <string>search</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
